Pipfile

```diff
@@ -45,11 +45,11 @@ datasets = ">=2.11.0, <3.0.0"
 neptune = "==1.3.1"
 Jinja2 = ">=3.1.2, <4.0.0"
 tenacity = ">=8.2.2, <9.0.0"
-h2o-wave = "0.26"
-tiktoken = "0.4.0"
-hf-transfer = "0.1.3"
-peft = "0.4.0"
-azure-storage-file-datalake = ">=12.12.0"
+h2o-wave = "==0.26.2"
+tiktoken = "==0.4.0"
+hf-transfer = "==0.1.3"
+peft = "==0.5.0"
+azure-storage-file-datalake = "==12.12.0"
```

Comment:
should be fine to leave as `>=` as this is a mature package which shouldn't change their API often.